packages/back-end/src/api/openapi/payload-schemas/PostMetricPayload.yaml

```diff
@@ -0,0 +1,158 @@
+type: object
+required:
+  - datasourceId
+  - name
+  - type
+properties:
+  datasourceId:
+    type: string
+    description:  ID for the [DataSource](#tag/DataSource_model)
+  owner:
+    type: string
+    description: Name of the person who owns this metric
+  name:
+    type: string
+    description: Name of the metric
+  description:
+    type: string
+    description: Description of the metric
+  type:
+    type: string
+    description: Type of metric. See [Metrics documentation](/app/metrics)
+    enum:
+      - binomial
+      - count
+      - duration
+      - revenue
+  tags:
+    type: array
+    description: List of tags
+    items:
+      type: string
+  projects:
+    type: array
+    description: List of project IDs for projects that can access this metric
+    items:
+      type: string
+  archived:
+    type: boolean
+  behavior:
+    type: object
+    required: []
+    properties:
+      goal:
+        type: string
+        enum:
+          - increase
+          - decrease
+      cap:
+        type: number
+        description: This should be non-negative
+      conversionWindowStart:
+        type: number
+        description: "The start of a Conversion Window relative to the exposure date, in hours. This is equivalent to the [Conversion Delay](/app/metrics#conversion-delay). <br/> Must specify both `behavior.conversionWindowStart` and `behavior.conversionWindowEnd` or neither."
+      conversionWindowEnd:
+        type: number
+        description: "The end of a [Conversion Window](/app/metrics#conversion-window) relative to the exposure date, in hours. This is equivalent to the [Conversion Delay](/app/metrics#conversion-delay) + Conversion Window Hours settings in the UI. In other words, if you want a 48 hour window starting after 24 hours, you would set conversionWindowStart to 24 and conversionWindowEnd to 72 (24+48). <br/> Must specify both `behavior.conversionWindowStart` and `behavior.conversionWindowEnd` or neither."
+      riskThresholdSuccess:
+        type: number
+        description: Threshold for Risk to be considered low enough, as a proportion (e.g. put 0.0025 for 0.25%). <br/> Must be a non-negative number.
+      riskThresholdDanger:
+        type: number
+        description: Threshold for Risk to be considered too high, as a proportion (e.g. put 0.0125 for 1.25%). <br/> Must be a non-negative number.
+      minPercentChange:
+        type: number
+        description: Minimum percent change to consider uplift significant, as a proportion (e.g. put 0.005 for 0.5%)
+      maxPercentChange:
+        type: number
+        description: Maximum percent change to consider uplift significant, as a proportion (e.g. put 0.5 for 50%)
+      minSampleSize:
+        type: number
+  sql:
+    type: object
+    description: "Preferred way to define SQL. Only one of `sql`, `sqlBuilder` or `mixpanel` allowed."
```

Comment:
Sorry to add one more comment about copy, but maybe we could say the following for each of these three fields:

> Only one of `sql`, `sqlBuilder` or `mixpanel` allowed, and at least one must be specified.